packages/edit-site/src/components/add-new-post/index.js

```diff
@@ -95,9 +95,10 @@ export default function AddNewPostModal( { postType, onSave, onClose } ) {
 			size="small"
 		>
 			<form onSubmit={ createPost }>
-				<VStack spacing={ 3 }>
+				<VStack spacing={ 4 }>
 					<TextControl
 						__next40pxDefaultSize
+						__nextHasNoMarginBottom
```

Comment:
## Testing instructions

Go to the Pages view in Site Editor (`wp-admin/site-editor.php?postType=page`), and click the Add New Page button.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/f005aca8-e454-4840-910c-1a15c2a0024d" alt="Add new page modal, before" width="411">|<img src="https://github.com/user-attachments/assets/977b8baa-3861-4d22-a88e-6b6276d442b8" alt="Add new page modal, after" width="409">|

@WordPress/gutenberg-design I'm seeing some inconsistency in the spacing before the CTA button in contexts like this. The [Figma](https://www.figma.com/design/e4tLacmlPuZV47l7901FEs/WordPress-Design-Library?node-id=73750-5178&t=DWq8qPlI9L1HWdW4-4) specs it as 24px — should I follow that everywhere? Also, would the spacing be different in a non-Modal context like Data Views Quick Edit, or a small block toolbar popover like the Language formatter?

<img src="https://github.com/user-attachments/assets/06eeb0fb-0cf6-443c-b3d2-489b2f820090" alt="Data Views Quick Edit" width="371">

<img src="https://github.com/user-attachments/assets/8c3e3e02-b2ec-4c95-8c35-650dc3a02554" alt="Language format popover" width="295">

Reply:
I think there could be a case for a dedicated `ModalFooter` (or similar) component that handles this. It should:

* Align the buttons (right)
* Set spacing
* Set the button order (primary always right-most)
* Limit to two buttons
* Handle small-screen behavior (make buttons full width and/or stack)

But that's probably part of a much broader update to `Modal`. Until we have a clear design for that I don't suppose the spacing matters too much – I'd probably just aim for what exists on trunk.

Reply:
Ok, if we don't have spec'ed values yet, I'm going to go with the path of least resistance as long as it's in the general vicinity of what's correct. In a lot of cases, that's going to be a space equidistant between all the control gaps (i.e. throw them all in a single `VStack`). This is going to be easier to deal with later once we decide on an official value.